spec/lib/tasks/invalid_activities_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+require "csv"
+Rails.application.load_tasks
+
+describe "invalid_activities.rake" do
+  it "creates a csv file on tmp folder with a list of invalid activities in RODA" do
+    activities = create_list(:project_activity, 5)
+    activities.first.update_columns(gdi: nil, collaboration_type: nil)
+    activities.last.update_columns(title: nil, programme_status: nil)
+
+    run_task(task_name: "invalid_activities")
+    invalid_activities_from_csv = CSV.read("tmp/invalid_activities.csv")
```

Comment:
Do we need to worry about ordering of the activities? I assume `CSV.read` retains the order so the first record is always the one we expect?

Reply:
That was my assumption too. When I run this on the console, with mock data it always kept activities in the same order.